task_reactredux/task1/src/actions/index.js

```diff
@@ -0,0 +1,40 @@
+export const onMailChange = (mail) => {
+  return {
+    type: 'ON_MAIL_CHANGE',
```

Comment:
move all strings to constants please, either one object `actionTypes` or const string for each action